hibernate-redis/src/main/java/org/hibernate/cache/redis/AbstractRedisRegionFactory.java

```diff
@@ -92,13 +94,21 @@ public boolean isMinimalPutsEnabledByDefault() {
         return true;
     }
 
+    protected void createJedisClientAndTimestamper(Settings settings, Properties properties) {
+        if (redis == null) {
```

Comment:
I guess so. The docs on RegionFactory say it will only be called once. I'd be a bit hesitant to take it at its word :)